### PR TITLE
✅ New decimals remainder test

### PR DESCRIPTION
**Pull request by JuaniRios**
## What?
- Add a test for checking a wide range of decimals for CTs work as intended.

## Why?
- We might have unexpected bugs when the decimals are too high or too low

## How?
- Similar to the community test `different_decimals_ct_works_as_expected`

### CODE REVIEW

**Comment by JuaniRios** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/Polimec/polimec-node/278?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#278** <a href="https://app.graphite.dev/github/pr/Polimec/polimec-node/278?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#277** <a href="https://app.graphite.dev/github/pr/Polimec/polimec-node/277?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#275** <a href="https://app.graphite.dev/github/pr/Polimec/polimec-node/275?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @JuaniRios and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by vstam1 — COMMENTED** [review]
Same comments as for previous prs

File: pallets/funding/src/tests/4_community.rs [hunk 1]
```diff
@@ -292,7 +292,7 @@ mod round_flow {
 			for decimals in 0..25 {
 				decimal_test(decimals);
 			}
-
+gt
```
Comment (vstam1) [inline pallets/funding/src/tests/4_community.rs:295]: ?

**Review by vstam1 — APPROVED** [review]

**Comment by JuaniRios** [comment]
### Merge activity

* **May 2, 5:12 AM EDT**: @JuaniRios started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/Polimec/polimec-node/278).

**Pull request merged**